### DISCUSSION TITLE
A standardised set of introductory slides

### DISCUSSION THREAD

**Discussion by karlgoldstraw** [nhsuk/nhsuk-service-manual#1253] [General]
# What 
Create a set of introductory slides for the NHS Digital design system that can form the basis of all talks or show and tells moving forward.

# Why 
To help familiarise people with the service manual and to underline existential reasons. 

## Resources

- There's a talk from Tim Paul on the Gov 
- [NHS service manual show and tells](https://service-manual.nhs.uk/whats-new/show-and-tells)

**Comment by sarawilcox** [comment]
This Slack thread: https://nhsuk.slack.com/archives/C0J62HNKX/p1621856528037200?thread_ts=1621855170.036900&cid=C0J62HNKX
